src/uninitialized_device.rs

```diff
@@ -94,6 +94,12 @@ impl<SpiBus: Bus> UninitializedDevice<SpiBus> {
         self.bus
             .write_frame(register::COMMON, register::common::MODE, &mode)?;
 
+        self.bus.write_frame(
+            register::COMMON,
+            register::common::SOCKET_INTERRUPT_MASK,
+            &[0xFF],
```

Comment:
We should get this as configurable data. People may have cases where they do not want the INTn line changing